src/compute/src/render/mod.rs

```diff
@@ -661,14 +663,15 @@ where
     fn rearrange_iterative<K, V>(
         &self,
         oks: Arranged<Child<'g, G, T>, TraceRowHandle<K, V, T, Diff>>,
+        name: &str,
     ) -> Arranged<G, TraceRowHandle<K, V, G::Timestamp, Diff>>
     where
         K: Columnation + ExchangeData + Hashable,
         V: Columnation + ExchangeData,
     {
         oks.as_collection(|k, v| (k.clone(), v.clone()))
             .leave()
-            .mz_arrange("Arrange export iterative")
+            .mz_arrange(name)
```

Comment:
Outside the scope of this PR: We're leaving some performance improvements on the floor because `as_collection` clones the time, which is a `Pointstamp` backed by a vector in this case. Next, we're calling `leave`, which calls `to_outer` on the timestamp, turning it back into a `Timestamp` a.k.a. `u64`. Combining the two could improve performance for WMR queries. Be warned, the `flat_map_ref`/`flat_map_batches` APIs don't give access to the time, so we'd need to roll our own. Do you mind recording this as an issue?

Reply:
Not at all, nice observation! Issue https://github.com/MaterializeInc/materialize/issues/22530.